src/fetch/transfer_file.py

```diff
@@ -22,7 +22,7 @@
     FILE_OUT_DIR,
     DOCS_URL,
 )
-from src.fetch.payouts import post_cip20_payouts
+from src.fetch.payouts import post_cip27_payouts
```

Comment:
In the near future I plan to factor out all pre-cip20 logic from this project and then we will not need this special method name anymore. Thanks for reminding me.

Reply:
In fact, if we rebased this branch on #311, we would not have to touch this code for a rename.